spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -190,6 +192,18 @@ public void testJson() throws Exception {
 	public static class Config {
 
 		@Bean
+		public static PropertyPlaceholderConfigurer ppc() {
```

Comment:
But yeah, that's minor and is just an info for the future to simplify your codding life :smile: 

I can polishing it that way on merge.
Just let me know